examples/sentence-transformers-training/sts/readme.md

```diff
@@ -0,0 +1,95 @@
+# Semantic Textual Similarity
+
+Semantic Textual Similarity (STS) assigns a score on the similarity of two texts. In this example, we use the [stsb](https://huggingface.co/datasets/sentence-transformers/stsb) dataset as training data to fine-tune our model. See the following example scripts how to tune SentenceTransformer on STS data:
+
+- **[training_stsbenchmark.py](training_stsbenchmark.py)** - This example shows how to create a SentenceTransformer model from scratch by using a pre-trained transformer model (e.g. [`distilbert-base-uncased`](https://huggingface.co/distilbert/distilbert-base-uncased)) together with a pooling layer.
+
+
+## Requirements
+
+First, you should install the requirements:
+```bash
+pip install -U sentence-transformers
+pip install git+https://github.com/huggingface/optimum-habana.git
+```
+
+
+## Usage
+
+To fine training on STS you can do following steps -
+
```

Comment:
Can you explicitly add the pre-training instruction for this task too?

Reply:
same as above

Reply:
same as above to make readmes consistent.